### DISCUSSION TITLE
My commits aren't rendered in the Commit History graphic!

### DISCUSSION THREAD

**Discussion by arsekil** [community/community#143531] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone,
To my biggest surprise, on my contributions graphicon most of my commits don't show up at all, even though in my repos' the commit are there for sure.

Did anyone else experience this kindof bugish behaviour? It's like I wouldn't have made commits, basically many of them are not showing up with a green dot.

**Comment by davevad93** [comment, marked as answer]
Hi @andrejmoltok  ,

Usually, when contributions don't appear in the activity graph, it's for the following reasons:

1. The email isn't properly configured in `git config`.
2. The commits have been pushed to a non-default branch (secondary branch) in which case the contributions will be counted when the changes are merged into the `main` branch via `pull request`.
3. The commits have been pushed to a forked repository, in which case the contributions will be counted when the changes are merged into the source repo via `pull request`.
4. The `Include private contributions in my profile` option is unchecked in the `Settings`.


For further reference, take a look at the docs:

-  [Why are my contributions not showing up on my profile?](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)

**Reply by arsekil** [reply]
Point number one reminded me of that I have changed my email address recently for my github account and I didn't realize that this would cause this 'issue'.

Thank you for clarifying this for me. Much appreciated 👍

**Comment by AyhemKouki** [comment]
Ensure that the email address in your Git commits matches the email address associated with your GitHub account. If the emails don’t match, GitHub won’t link those commits to your account, and they won’t appear on your contributions graph. You can verify and correct this in Git by running:
git config user.email
Update it if you want to make sure there is no problem related to the email 
git config user.email "your_email@example.com"
